setup.py

```diff
@@ -35,7 +35,7 @@
 
 INSTALL_REQUIRES = [
     'numpy >= 1.12',
-    'pandas >= 0.23.0, <1.0.0',
+    'pandas >= 0.23.0, <=1.0.0',
```

Comment:
```suggestion
    'pandas >= 0.23.0, <1.0.0, >=1.0.2',  # exclude 1.0.0 & 1.0.1 for GH142
```
Pandas 1.0.2 was released yesterday with the fix for this issue.  Should we forget the workaround and just exclude 1.0.0 and 1.0.1 like this?  I'd guess that people will move on from 1.0.0 and 1.0.1 pretty quickly, so IMHO it's not worth maintaining weird code to support those versions.

Reply:
I agree and support requiring 1.0.2